packages/app/components/Settings/CourseAdminPanel.tsx

```diff
@@ -0,0 +1,88 @@
+import { BellOutlined, EditOutlined } from "@ant-design/icons";
+import { Col, Menu, Row, Space } from "antd";
+import Link from "next/link";
+import React, { ReactElement, useState } from "react";
+import styled from "styled-components";
+import { useProfile } from "../../hooks/useProfile";
+import CourseOverrideSettings from "./CourseOverrideSettings";
+import { SettingsPanelAvatar } from "./SettingsSharedComponents";
+import TACheckInCheckOutTimes from "./TACheckInCheckOutTimes";
+
+export enum CourseAdminOptions {
+  CHECK_IN = "CHECK_IN",
+  OVERRIDES = "OVERRIDES",
+}
+
+interface CourseAdminPageProps {
+  defaultPage: CourseAdminOptions;
+  courseId: number;
+}
+
+const VerticalDivider = styled.div`
+  @media (min-width: 767px) {
+    border-right: 1px solid #cfd6de;
+    margin-right: 32px;
+  }
+`;
+
+const CenteredText = styled.p`
+  text-align: center;
+`;
+
+export default function CourseAdminPanel({
+  defaultPage,
+  courseId,
+}: CourseAdminPageProps): ReactElement {
+  const profile = useProfile();
+  const [currentSettings, setCurrentSettings] = useState(
+    defaultPage || CourseAdminOptions.CHECK_IN
+  );
+
+  return (
+    <Row>
+      <Col span={4} style={{ textAlign: "center" }}>
+        <SettingsPanelAvatar />
+        <CenteredText>
+          Welcome back {profile?.firstName} {profile?.lastName}
```

Comment:
oops I was thinking of adding like "you last checked this page on X date" using LocalStorage... maybe still a good idea?